README.md

```diff
@@ -61,46 +60,13 @@ A configured example is located in the [config.sample.yaml](./config.samples.yam
   change the icon to gently remind you to connect.
 
 
-### Oauth token configuration
-
-Before launching the applet you need to first generate a Google OAuth 2.0 Client
-Credentials and authorize it.
-
-* Go to the Google Cloud Console: https://console.developers.google.com/apis/credentials
-
-* Select Create credentials and Oauth Client ID
-
-![Create Credentials image](./.github/screenshot/create-oauth-1.png)
-
-* Select "Desktop app" for the Application Type and give it a name.
-
-![Create Desktop app](./.github/screenshot/create-oauth-2.png)
-
-* On the next screen you can just click OK, since we will look over the auth json file.
-
-* Now next to your created application you will see an icon to download the auth
- json file, click on it and store it somewhere in a name you can rememeber (ie: `client_secret.json`)
-
-![Click on download file](./.github/screenshot/create-oauth-2.png)
-
-* In the terminal launch `gnome-next-meeting-applet-auth` with the path to the auth json file : 
-
-   `gnome-next-meeting-applet-auth ~/Downloads/client_secret.json`
-
-* This will launch your webbrowser to do the oauth dance
-
-* If your browser tells you the apps is unverified you can say it's okay (it's
- should be the one that you have created yourself).
-
-* After the last click of authorizations the credentials has been created.
-
-![Created](./.github/screenshot/create-oauth-5.png)
+### Calendars
 
-* You can now launch the gnome-next-meeting applet.
+All calendars are configured directly in Gnome Online Account setting, it will grab it from there.
 
 ### Starting
 
-You can just do : 
+You can just do :
 
 Alt+F2 and start `gnome-next-meeting-applet` make sure you have done the oauth
```

Comment:
We don't need to dance anymore here :dancers:

Reply:
You are right thanks ! 🎯